client/my-sites/email/titan-setup-thank-you/style.scss

```diff
@@ -1,3 +1,8 @@
 .titan-setup-thank-you__icon-external {
 	margin-left: 4px;
 }
+
+.layout__content {
```

Comment:
This rule needs to be scoped better. It will apply to the whole UI once this CSS has been loaded, so we can't target just `.layout__content` without additional selectors.

Reply:
Dale, I am struggling to get a better selector. We need the content layout to have this rule when the thank you page is added, so no borders are shown in the thank you page component.

Reply:
Hmm... I think we may be able to work around the left and right padding by adding negative margins to the Thank You container. To be specific, at desktop width, it looks like `margin-left: -33px` and `margin-right: 32px` could work.

I haven't had something that has worked to fix the height/bottom area yet, but will keep thinking about it.

Reply:
That's a clever idea, I didn't think about it. Also I missed this comment 🤦